iphone/Classes/TiMediaVideoPlayerProxy.h

```diff
@@ -45,6 +49,7 @@
 @property (nonatomic, readwrite, assign) TiColor *backgroundColor;
 @property (nonatomic, readonly) NSNumber *playing;
 @property (nonatomic, copy) NSNumber *volume;
+@property (nonatomic, readwrite, assign) id pictureInPictureEnabled;
```

Comment:
Should be `NSNumber` if public API. `BOOL` otherwise.

Reply:
Made it NSNumber.